src/validate.ts

```diff
@@ -19,6 +19,10 @@ export const validate = (
   configPaths: string[],
   { schemaPath }: Options
 ): true => {
+  if (schemaPath === null) {
```

Comment:
What if `schemaPath` were `undefined` instead of `null`?

Reply:
i'll play around with this a bit more but i'd say that json-schema catches this so that `schemaPath` can only be a string or `null`.

Reply:
ok, could be right. still seems safer to just make this a standard JS falsy check:

```suggestion
  if (schemaPath) {
```

Reply:
empty string is basically like path to non-existing file. Both is caught few lines below already.